documentation/1.1/content/release notes/_index.md

```diff
@@ -1,29 +1,76 @@
-- Added these actions:
-   - Validate WDT model files
-   - Create and push an auxiliary image
-   - Update and uninstall the WebLogic Kubernetes Operator
-   - Undeploy the WebLogic domain
-   - Uninstall the ingress controller
+#### Major New Features
+- Updated macOS environment control to specify extra environment settings to pass down to Docker/Podman, kubectl, and Helm invocations.
+- Bundling WebLogic Deploy Tooling 2.0.0. See [WDT release notes](https://github.com/oracle/weblogic-deploy-tooling/releases/tag/release-2.0.0) for details.
+- New Validate Model action that leverages WDT 2.0 changes to validate the model files and reports any validation errors it finds.
+- Updated Prepare Model action that leverages WDT 2.0 changes to improve model preparation for deploying with WebLogic Kubernetes Operator.
+- Added support for splitting the FMW (WebLogic or FMW Infrastructure) and WDT model images (aka. auxiliary images).
+  - New Create Auxiliary Image action to create an auxiliary image.
+  - New Push Auxiliary Image action to push an auxiliary image to an image registry.
+- New Update Operator action to apply changes such as a version upgrade to an existing WebLogic Kubernetes Operator installation.
+- New Uninstall Operator action to uninstall the WebLogic Kubernetes Operator installation and optionally, its Kubernetes namespace.
+- New Undeploy Domain action to undeploy an existing domain  and optionally, its Kubernetes namespace.
+- New Uninstall Ingress Controller action to uninstall the ingress controller installation and optionally, its Kubernetes namespace.
+- Updated Ingress Route editing to improve control for SSL and provide lists of available target services and target ports.
 
-### Known Issues
+#### Other Changes
+- Added the `Save As` menu item to allowing saving a project and any model files to a new name/location.
+- Improved Internet connectivity check mechanism and added configurable timeout settings to address issues some users were
+  having with the default OCI DNS settings causing extra long DNS name resolution times that were triggering the hard-coded timeout.
+- Upgraded Electron to version 16.
+- Upgraded the Model Code View page's embedded ACE editor to 1.4.13.
+- Relocated the older, macOS-specific `Extra Path Directories` functionality from the Kubernetes Client Configuration page to the
+  Project Settings page to better align with new macOS environment settings and the scope to which they apply.
+- Replaced `Help` menu's `Visit Website` menu item with a `View Documentation` menu item that opens the WebLogic Kubernetes Toolkit UI
+  documentation is the user's browser.
+- Added a result dialog to display the Primary Image's custom base image's content found by the Inspect Image action.
+- Added a warning message to the Discover Model action if the project has existing model files that will be overwritten by the action.
+- Changed the Windows and Linux About dialog to not be modal.
```

Comment:
About dialog -> `About` dialog (I suggest using code font here for the name of this dialog to distinguish it from reading as the "Linux About" dialog.)